app/routes/index.js

```diff
@@ -35,5 +35,7 @@ module.exports = function init(app) {
   app.use('/v1/campaigns', campaignsIndexRoute);
 
   // Receives inbound message from Gambit Conversations service.
-  app.use('/v1/receive-message', receiveMessageRoute);
+  app.use('/v1/campaignActivity', campaignActivityRoute);
+  // TODO: Remove this once we make the switch in Conversations.
+  app.use('/v1/receive-message', campaignActivityRoute);
```

Comment:
Minor, but we could declare this route in an array as per the [docs](https://expressjs.com/en/4x/api.html#path-examples):

```
app.use(['/v1/campaignActivity', '/v1/receive-message'], campaignActivityRoute);
```